integration/addition_gids_test.go

```diff
@@ -31,7 +31,7 @@ import (
 )
 
 func TestAdditionalGids(t *testing.T) {
-	testPodLogDir, err := os.MkdirTemp("/tmp", "additional-gids")
+	testPodLogDir, err := os.MkdirTemp("", "additional-gids")
```

Comment:
t.TempDir is more preferable, but can be another PR

Reply:
Sure. This at-least lines them up with the other `os.MkdirTemp calls that didn't have `/tmp` hardcoded.